apps/knockdog/src/widgets/map-with-schools/ui/MapWithSchools.tsx

```
@@ -0,0 +1,105 @@
+import Link from 'next/link';
+import { useSearchParams } from 'next/navigation';
+import { cn } from '@knockdog/ui/lib';
+import { Float, Icon } from '@knockdog/ui';
+import { ListFAB, CurrentLocationFAB, Map, RefreshFAB } from '@features/map';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)